Replicator/Inserter.cc

```diff
@@ -112,10 +112,12 @@ namespace litecore { namespace repl {
             if (rev->flags & kRevPurged) {
                 // Server says the document is no longer accessible, i.e. it's been
                 // removed from all channels the client has access to. Purge it.
-                if (_db->insertionDB().useLocked()->purgeDocument(rev->docID))
-                    logVerbose("    {'%.*s' removed (purged)}", SPLAT(rev->docID));
+                auto locked = _db->insertionDB().useLocked();
+                if (collection()->purgeDocument(rev->docID)) {
```

Comment:
Or simpler, _db->insertionDB()-useCollection(collection())->purgeDocument

Reply:
insertionDB() is AccessLockedDB (not DBAccess) so it doesn't have useCollection().

Reply:
You're right. Disregard it.